### ISSUE TITLE
[WEC-Sim Applications]MoorDyn v2 error 

### ISSUE THREAD

**Issue by hachikoi1** [WEC-Sim/WEC-Sim#1178]
**Note: italicized text below is include as an example and should be updated before submission. If you feel any section is not applicable to your request, please replace with 'N/A' rather than delete a section.**

**Cite the [WEC-Sim applications](https://github.com/WEC-Sim/WEC-Sim_Applications) case in question**
_Please identify the WEC-Sim applications case in question._

**Describe the bug or issue**
_A clear and concise description of what the perceived bug or issue is._
Hi [jtgrasb](https://github.com/jtgrasb)
I'm confused about the input file for WEC-Sim V6.0 - MoorDyn V2. I made the input file with reference to \applications\Mooring\MoorDyn\Mooring\lines.txt, but the lines.log always report a bug: **ERR ../../source/Line.cpp:759 getStateDeriv(): NaN detected**. 

I changed the parameters, but the error persists. And as a result, WEC-Sim reports an error similar to https://github.com/WEC-Sim/WEC-Sim/issues/1167#issue-1993914228. 
[Test.zip](https://github.com/WEC-Sim/WEC-Sim/files/13561509/Test.zip) is my studied cases, the test model of UMaine VolturnUS-S semi-submersible platform. The information of 1:70 scale model can be found [here](https://www.ccp-wsi.ac.uk/data_repository/test_cases/test_case_015), and 1:1 model information is in [Definition of the UMaine VolturnUS-S Reference.pdf](https://github.com/WEC-Sim/WEC-Sim/files/13561451/Definition.of.the.UMaine.VolturnUS-S.Reference.pdf). Could you check what's wrong with the settings?

Many thanks

Tuanyuan

**To Reproduce**
_Steps to reproduce the behavior:_

1. _Go to '...'_
1. _Click on '....'_
1. _Scroll down to '....'_
1. _See error_

**Expected behavior**
_A clear and concise description of what you expected to happen._

**Screenshots**
_If applicable, add screenshots to help explain your problem. For example, a screen shot of the exact error message from the MATLAB command line._

**Workstation Information (this section MUST BE completed):**
 - OS: [e.g. iOS8.1, Windows 10, Linux 5.12.1]
 - [MATLAB/Simulink Release](https://www.mathworks.com/help/matlab/release-notes.html) [2019b, 2020a, 2020b, 2021a]
 - [WEC-Sim Release](http://wec-sim.github.io/WEC-Sim/man/release_notes.html) [v3.1, v4.0, v4.1]

**Additional context**
_Add any other context about the problem here._

**Comment by hachikoi1** [comment]
Hi jtgrasb
The updated input file according to  PR #54 is shown below. 
```
----------------------------------------------------------- MoorDyn INPUT FILE
1:70 scale model of the IEA 15MW reference wind turbine (IEA-15-240-RWT) and Uchaine VolturnUS-S semi-submersible platform https://www.ccp-wsi.ac.uk/data_repository/test_cases/test_case_015
----------------------------------------------------------- LINE TYPES
TypeName       Diam           MassDen        EA             BA/-zeta       EI             Cd             Ca             CdAx           CaAx
(name)         (m)            (kg/m)         (N)            (N-s/-)        (N-m^2)        (-)            (-)            (-)            (-) 
Chain          0.0048         0.144          2.20E+06      -1.000          0.000          2.000          0.820          0.400          0.270
----------------------------------------------------------- BODIES 
ID             Attachment     X0             Y0             Z0             r0             p0             y0             Mass           CG*            I*             Volume         CdA*           Ca
(#)            (-)            (m)            (m)            (m)            (deg)          (deg)          (deg)          (kg)           (m)            (kg-m^2)       (m^3)          (m^2)          (-)
1              Coupled       -0.0362        -0.0032        -0.1466         0             -1.0333         0              0.0            0              0              0              0              0
----------------------------------------------------------- POINTS                                           
ID             Attachment     X              Y              Z              M              V              CdA            CA
(#)            (word/ID)      (m)            (m)            (m)            (kg)           (m^3)          (m^2)          (-)
1              Fixed         -9.52500        0.00000       -2.86000        0.000          0.000          0.000          0.000
2              Fixed          4.41200       -7.65500       -2.86000        0.000          0.000          0.000          0.000
3              Fixed          4.41200        7.65500       -2.86000        0.000          0.000          0.000          0.000
4              Vessel        -0.83443        0.00000       -0.17798        0.000          0.000          0.000          0.000
5              Vessel         0.42437       -0.72680       -0.17798        0.000          0.000          0.000          0.000
6              Vessel         0.42437        0.72680       -0.17798        0.000          0.000          0.000          0.000
----------------------------------------------------------- LINES                                         
ID             LineType       AttachA        AttachB        UnstrLen       NumSegs        LineOutputs
(#)            (-)            (#)            (#)            (m)            (-)            (-)
1              Chain          1              4              9.685          50             tp
2              Chain          2              5              9.017          50             tp
3              Chain          3              6              9.017          50             tp
----------------------------------------------------------- SOLVER OPTIONS
0.0005                                                      dtM             - time step to use in mooring integration (s)
RK2                                                         tScheme         - time integrator
2.86                                                        WtrDpth         - water depth
9.80665                                                     g               - gravity acceleration (m/s^2)
998.2072                                                    rho             - water density (kg/m^3)
1.0                                                         dtIC      		- time interval for analyzing convergence during IC gen (s)
200.0                                                       TmaxIC    		- max time for ic gen (s)
0.001                                                       threshIC  		- threshold for IC convergence (-)   
5.0                                                         CdScaleIC 		- factor by which to scale drag coefficients during dynamic relaxation (-)
0                                                           WaveKin         - wave kinematics flag (0=neglect, the only option currently supported)
3.0E+06                                                     kbot            - bottom stiffness (Pa/m)
3.0E+05                                                     cbot      		- bottom damping (Pa-s/m)
2                                                           writeLog        - write a log file
1                                                           WriteUnits      - write the units header on the output files
----------------------------------------------------------- OUTPUTS                                  
FairTen1                                           
FairTen2                                            
FairTen3                                          
AnchTen1                                            
AnchTen2                                          
AnchTen3
-----------------------------------------------------------                          

```

**Comment by hachikoi1** [comment]
Hi jtgrasb
The updated input file according to PR #54 is shown below.

```
----------------------------------------------------------- MoorDyn INPUT FILE
1:70 scale model of the IEA 15MW reference wind turbine (IEA-15-240-RWT) and UMaine VolturnUS-S semi-submersible platform https://www.ccp-wsi.ac.uk/data_repository/test_cases/test_case_015
----------------------------------------------------------- LINE TYPES
TypeName       Diam           MassDen        EA             BA/-zeta       EI             Cd             Ca             CdAx           CaAx
(name)         (m)            (kg/m)         (N)            (N-s/-)        (N-m^2)        (-)            (-)            (-)            (-) 
Chain          0.0048         0.144          2.20E+06      -1.000          0.000          2.000          0.820          0.400          0.270
----------------------------------------------------------- BODIES 
ID             Attachment     X0             Y0             Z0             r0             p0             y0             Mass           CG*            I*             Volume         CdA*           Ca
(#)            (word)         (m)            (m)            (m)            (deg)          (deg)          (deg)          (kg)           (m)            (kg-m^2)       (m^3)          (m^2)          (-)
1              Coupled       -0.0362        -0.0032        -0.1466         0             -1.0333         0              0.0            0              0              0              0              0
----------------------------------------------------------- POINTS                                           
ID             Attachment     X              Y              Z              M              V              CdA            CA
(#)            (word/ID)      (m)            (m)            (m)            (kg)           (m^3)          (m^2)          (-)
1              Fixed         -9.52500        0.00000       -2.86000        0.000          0.000          0.000          0.000
2              Fixed          4.41200       -7.65500       -2.86000        0.000          0.000          0.000          0.000
3              Fixed          4.41200        7.65500       -2.86000        0.000          0.000          0.000          0.000
4              Vessel        -0.83443        0.00000       -0.17798        0.000          0.000          0.000          0.000
5              Vessel         0.42437       -0.72680       -0.17798        0.000          0.000          0.000          0.000
6              Vessel         0.42437        0.72680       -0.17798        0.000          0.000          0.000          0.000
----------------------------------------------------------- LINES                                         
ID             LineType       AttachA        AttachB        UnstrLen       NumSegs        LineOutputs
(#)            (name)         (ID)           (ID)           (m)            (-)            (-)
1              Chain          1              4              9.685          50             tp
2              Chain          2              5              9.017          50             tp
3              Chain          3              6              9.017          50             tp
----------------------------------------------------------- SOLVER OPTIONS
0.0005                                                      dtM             - time step to use in mooring integration (s)
RK2                                                         tScheme         - time integrator
2.86                                                        WtrDpth         - water depth
9.80665                                                     g               - gravity acceleration (m/s^2)
998.2072                                                    rho             - water density (kg/m^3)
1.0                                                         dtIC      		- time interval for analyzing convergence during IC gen (s)
200.0                                                       TmaxIC    		- max time for ic gen (s)
0.001                                                       threshIC  		- threshold for IC convergence (-)   
5.0                                                         CdScaleIC 		- factor by which to scale drag coefficients during dynamic relaxation (-)
0                                                           WaveKin         - wave kinematics flag (0=neglect, the only option currently supported)
3.0E+06                                                     kbot            - bottom stiffness (Pa/m)
3.0E+05                                                     cbot      		- bottom damping (Pa-s/m)
2                                                           writeLog        - write a log file
1                                                           WriteUnits      - write the units header on the output files
----------------------------------------------------------- OUTPUTS                                  
FairTen1                                           
FairTen2                                            
FairTen3                                          
AnchTen1                                            
AnchTen2                                          
AnchTen3
-----------------------------------------------------------                          

```

**Comment by RyanDavies19** [comment]
Note that this issue is now being addressed in FloatingArrayDesign/MoorDyn#169
